decentraland/unity-renderer
# refactor: biw refactor

## What does this PR change?
Fixes #706 

This is a part of the refactor of builder in world where we get out the prefab from the initial scene and we only instantiate it if the feature is active

## How to test the changes?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:refactor/biw
2. Enter builder in world 
3. Check that everything is worked as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
